tests/e2e/specs/e2e.spec.ts

```diff
@@ -189,6 +189,18 @@ test("body attributes are copied over", async ({ page }) => {
   expect(await body.getAttribute("data-other")).toEqual("false");
 });
 
+test("uses config body attributes", async ({ page }) => {
+  await page.goto("./vite-page");
+  const body = page.locator("body").last();
+  expect(await body.getAttribute("id")).toEqual("test-id");
+  expect(await body.getAttribute("class")).toEqual("test-classname");
+
+  await page.goto("./body-test");
```

Comment:
need to navigate via clicking a link (goto is like typing in a url, and so it will not go through Vike's navigation system). I would add a link on body-test to go to vite-page, and then test going to vite-page and clicking the link